Enterprise-CMCS/macpro-onemac
# Oy2 22277 - split TEs into two types for 1915c and 1915b

Story: https://qmacbis.atlassian.net/browse/OY2-22277
Endpoint: See github-actions bot comment

### Details

Temporary extensions are divided into types based on selection of 1915c. or 1915b on the sumbission form. They still share the same UI form and therefore same api submission endpoint. At point of submission the api will determine what configuration type to submit based on the selected temporaryExtensionType.

This change also is migrating existing temporary extensions to their proper type. This is done in two parts. One from a list of TEs that have been manually mapped (those that existed before package release and therefore do not have a temporaryExtensionType attribute) and the other for all TEs submitted after package release and the type is present and therefore can be used to udpate componentType to the proper value.

### Changes

- I think the test coverage diff is so low because of the migration script. Do we care?

### Test Plan

1. login as state submitter
2. submit new temporary extension using type 1915(b)
3. verify new package on dashboard shows appropriate type label
4. repeat for 1915(c)
